src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -513,10 +514,27 @@ protected override void Generate(RenameTableOperation operation, [CanBeNull] IMo
                     .Append(method);
             }
 
-            builder
-                .Append(" (")
-                .Append(ColumnList(operation.Columns))
-                .Append(")");
+            if (operators == null) {
+                builder
+                    .Append(" (")
+                    .Append(ColumnList(operation.Columns))
+                    .Append(")");
+            }
+            else {
+                var operatorList = operators.Split(' ');
+                var columnList = new List<string>();
+                for (var k = 0; k < operation.Columns.Count - 1; ++k) {
+                    var columnWithOperator = ColumnList(operation.Columns[k]);
+                    if (k < operatorList.Count) {
+                        columnWithOperator += ' ' + operatorList[k];
+                    }
+                    columnList.Add(columnWithOperator);
+                }
+                builder
```

Comment:
This code is duplicated above, for the case where no operators are defined. Set up your column list (separately for each case), then render once.

Reply:
I can see in EF sources that ColumnList function transforms each column name and join them. But I can't find a function how to transform each column name individually. That is why I use ColumnList to transform each column name in secon case ( and that is not the optimal way) that is why I duplicated the code. What can you recommend me?

Reply:
I was just thinking about generating the contents inside the parentheses differently (based on the condition), and then having common code which appends the parentheses and the contents (nothing fancy).